README.md

```diff
@@ -40,20 +40,27 @@ $ sudo ./start.sh #run this each time
 Then, still on your development PC, head to rovercode.org and connect to your "rover" (your PC running the service).
 
 #### Development PC Alternate Setup (Docker)
-Rather use Docker? We have an image for you! First, on your development PC:
+Rather use Docker? First, on your development PC:
 ```bash
 $ sudo apt install git docker.io
 $ git clone --recursive https://github.com/aninternetof/rovercode.git && cd rovercode
-$ sudo docker pull cabarnes/rovercode
-$ sudo docker run --name rovercode -v $PWD:/var/www/rovercode -p 80:80 -d cabarnes/rovercode
+$ sudo docker build -t rovercode .
```

Comment:
Yeah, I think so. I just didn't know if it was still getting updated with the move to store.docker, so I updated this to definitely work for now.